.circleci/build-wheel.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
```

Comment:
We now have like 3 copies of roughly the same script, should we be consolidating?